src/Sentry.Unity/SentryUnitySDK.cs

```diff
@@ -56,6 +56,13 @@ private SentryUnitySdk(SentryUnityOptions options)
                         new ScreenshotAttachmentContent(options, SentryMonoBehaviour.Instance))));
             }
 
+            if (options.AttachViewHierarchy)
+            {
+                SentrySdk.ConfigureScope(s =>
```

Comment:
A problem with this approach is that `scope.Clear()` will get rid of it

Reply:
That's true. Same for screenshots. I already talked to @mattjohnsonpint about getting `hints` into the .NET SDK next. And then both, screenshots and view hierarchy get their own event processor. Or so the idea.